src/platform/nrfconnect/CHIPDevicePlatformConfig.h

```diff
@@ -47,6 +47,10 @@
 #define CHIP_DEVICE_CONFIG_USE_TEST_SETUP_DISCRIMINATOR CONFIG_CHIP_DEVICE_DISCRIMINATOR
 #endif
 
+#ifndef CHIP_DEVICE_CONFIG_FACTORY_DATA_VERSION
+#define CHIP_DEVICE_CONFIG_FACTORY_DATA_VERSION CONFIG_CHIP_FACTORY_DATA_VERSION
```

Comment:
`CHIP_DEVICE_CONFIG...` constants are needed in platform-independent code. In platform-specific code we can use Kconfig constants directly.